compiler/src/yul/mappers/expressions.rs

```diff
@@ -67,8 +71,7 @@ fn move_expression(
     from: Location,
     to: Location,
 ) -> Result<yul::Expression, CompileError> {
-    let typ =
-        FixedSize::try_from(typ).map_err(|_| CompileError::static_str("invalid attributes"))?;
+    let typ = to_fixed_size(&typ)?;
```

Comment:
I think we should keep the original line with an `expect` instead of `map_err`, so we crash in this situation. All the results should be removed in this module.